chain/network/src/types.rs

```diff
@@ -782,6 +782,37 @@ pub struct NetworkConfig {
     pub most_weighted_peer_horizon: u128,
     /// Period between pushing network info to client
     pub push_info_period: Duration,
+    /// Peers on blacklist by IP:Port.
+    /// Nodes will not accept or try to establish connection to such peers.
+    pub blacklist: Vec<PatternAddr>,
```

Comment:
Do you think it might be beneficial to have it as a hashmap indexed by ip address?

Reply:
If this list grows it will be certainly beneficial but this was intended to use on test (with small blacklist). Since this can probably change I'll make the update.

Reply:
done